src/graphics/render_window.rs

```diff
@@ -328,6 +477,7 @@ impl RenderWindow {
     ///
     /// Return the position in pixels
     ///
+    /// # Usage Example
```

Comment:
There is a stray Usage Example header here, without example

Reply:
Done.